### PR TITLE
chore: journeys all modern

### PR DESCRIPTION
**Pull request by mikeallisonJS**
## Summary by CodeRabbit

* **New Features**
  * Journey duplication, publish/feature/archive/delete/restore/trash actions
  * QR code short-links and management
  * Host, team, user-team, and team-invite management
  * Journey collections, user invites, journey access requests/approvals
  * Admin Power BI embeds for reports
  * Growth Spaces integration create/update

* **Enhancements**
  * Cursor-based pagination for journey events and visitors
  * Journey customization field parsing and editable fields
  * Journey theme, notification, and email preference management
  * Visitor update endpoints and improved ACL-aware queries

* **Other**
  * Journey lookup defaults to slug for idType

### CODE REVIEW

File: apis/api-journeys-modern/src/schema/qrCode/qrCode.service.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    customDomain.name !== hostname
+  ) {
+    throw new GraphQLError('Invalid hostname', {
+      extensions: { code: 'BAD_USER_INPUT' }
+    })
+  } else if (origin !== JOURNEYS_URL) {
+    throw new GraphQLError('Invalid hostname', {
+      extensions: { code: 'BAD_USER_INPUT' }
+    })
+  }
```
[the remark this answers is not in the document]
Comment (mikeallisonJS) [reply apis/api-journeys-modern/src/schema/qrCode/qrCode.service.ts:201]: Keeping as-is for parity. This `parseAndVerifyTo` is a direct port of the existing implementation in `apis/api-journeys/src/app/modules/qrCode/qrCode.service.ts` (on `main`), which uses the identical `else if (origin !== JOURNEYS_URL)` control flow. This PR is the api-journeys → api-journeys-modern migration (chore: journeys all modern), where behavioral parity with the existing API is the goal.

You're right that the `else if` also runs on the valid custom-domain path (`customDomain.name === hostname`), but that behavior already exists in `main` and changing it only here would diverge the two implementations mid-migration. If this edge case warrants a fix, it should be addressed in both `api-journeys` and `api-journeys-modern` together — with test coverage added to both — as a separate change rather than introducing a behavioral difference during the port.

**Comment by mikeallisonJS** [comment]
## Review feedback addressed (cc74e4191)

One unresolved thread (CodeRabbit) on `apis/api-journeys-modern/src/schema/qrCode/qrCode.service.ts`.

**Challenged (resolved — no code change):**
- `qrCode.service.ts:197-201` — CodeRabbit flagged that the `else if (origin !== JOURNEYS_URL)` branch also runs on the valid custom-domain path. The observation is technically correct, but this `parseAndVerifyTo` is a **direct port** of the existing implementation in `apis/api-journeys/.../qrCode.service.ts` (verified identical to `main`), which uses the same control flow. This PR is the api-journeys → api-journeys-modern migration, where behavioral parity with the existing API is the goal — changing validation only here would diverge the two implementations mid-migration. If the edge case warrants a fix, it should be done in both APIs together with test coverage, as a separate change. CodeRabbit concurred with this rationale in-thread.

**Review by csiyang — DISMISSED** [review]

**Review by csiyang — APPROVED** [review]
